schema/defs-image.json

```diff
@@ -4,7 +4,7 @@
     "mediaType": {
       "id": "https://opencontainers.org/schema/image/mediaType",
       "type": "string",
-      "pattern": "^[a-z]+/[0-9a-zA-Z.+]+$"
+      "pattern": "^[A-Za-z0-9][A-Za-z0-9!#$&-^_.+]{0,126}/[A-Za-z0-9][A-Za-z0-9!#$&-^_.+]{0,126}$"
```

Comment:
@stevvooe enum seems too strict, which will lead customized types failed during verifying. I checked `RFC 6838`, the pattern really follows it. I think `RFC 6838`  rule should be OK here.